### ISSUE TITLE
problem with version 0.6.2 

### ISSUE THREAD

**Issue by camilokawerin** [spatools/grunt-html-build#82]
Hi. I've cloned the repository and tried to process some files I've been processing earlier with the version 0.5.3 and the result files are exactly the same as the source one. I couldn't figure out which change has broken it, but if I replace the whole `build-html.js` by the one from the 0.5.3 version, everything goes fine.

**Comment by camilokawerin** [comment]
Installing the package with npm is the same thing.

**Comment by SomaticIT** [comment]
It may be a problem with line ending. Please try to use the line ending corresponding to your OS.
I will try to fix this issue in future releases.

**Comment by camilokawerin** [comment]
If you give me some idea about what part of the code should be fixed, I can take a look 😄

**Comment by SomaticIT** [comment]
if you take a look at the `build-html.js`, you will see that we use the `grunt.util.linefeed` [L32](https://github.com/spatools/grunt-html-build/blob/master/tasks/build-html.js#L32) to determine the EOL of the file depending on your OS.

The matching system use this EOL to replace tags in your input file so if `grunt.util.linefeed`=`\r\n` and your file's EOL=`\n` it will never match tags and never replace them.

We need to improve this behavior :
1. We need to determine the input file's EOL (before transforming content [L379](https://github.com/spatools/grunt-html-build/blob/master/tasks/build-html.js#L379))
2. We need to ensure the input file's EOL is valid for all lines (before transforming content [L379](https://github.com/spatools/grunt-html-build/blob/master/tasks/build-html.js#L379))

The best way should be to create a new function `ensureContent` which find and ensure file's EOL then change the line [L379](https://github.com/spatools/grunt-html-build/blob/master/tasks/build-html.js#L379) by :

```javascript
content = grunt.file.read(src);
content = ensureContent(content, params);
content = transformContent(content, params, dest);

//...
function ensureContent(content, params) {
    params.EOL = //find content EOL
    return content.replace(/\r?\n/g, params.EOL);
}
```

Finally all occurrences of `EOL` should be replaced by `params.EOL` or `options.EOL`.

Thanks for your contribution.

**Comment by SomaticIT** [comment]
Hi everyone,
I'm glad to announce that this issue is finally fixed in release `0.7.0`.
You can now update your `grunt-html-build` package to fix this issue.

This release introduces a new `EOL` option which can be provided by target.
If this option is not provided, the module will automatically detect `EOL` from specified input file.

Hope it helps.
Thanks

**Comment by camilokawerin** [comment]
Great! Thank you @SomaticIT
